sdk/src/metrics/state/temporal_metric_storage.cc

```diff
@@ -88,20 +91,20 @@ bool TemporalMetricStorage::buildMetrics(CollectorHandle *collector,
     if (aggregation_temporarily == AggregationTemporality::kCumulative)
     {
       // merge current delta to previous cumulative
-      last_aggr_hashmap->GetAllEnteries([&merged_metrics, this](const MetricAttributes &attributes,
-                                                                Aggregation &aggregation) {
-        auto agg = merged_metrics->Get(attributes);
-        if (agg)
-        {
-          merged_metrics->Set(attributes, agg->Merge(aggregation));
-        }
-        else
-        {
-          merged_metrics->Set(
-              attributes, DefaultAggregation::CreateAggregation(instrument_descriptor_, nullptr));
-        }
-        return true;
-      });
+      last_aggr_hashmap->GetAllEnteries(
+          [&merged_metrics, this](const MetricAttributes &attributes, Aggregation &aggregation) {
```

Comment:
Is this jus reformat?

Reply:
yes, this is done by clang-format. The only changes are lines 103, and 104 below. To handle the `else` scenario when there are no measurements recorded since the last collection.